pkg/cloudprovider/kubevirt/loadbalancer.go

```diff
@@ -323,6 +346,39 @@ func (lb *loadbalancer) getLoadBalancerCreatePollInterval() time.Duration {
 	return defaultLoadBalancerCreatePollInterval
 }
 
+func (lb *loadbalancer) filterNodes(service *corev1.Service, nodes []*corev1.Node) ([]*corev1.Node, error) {
```

Comment:
This works well but we need to add a Watch of the Pods to re-apply the `filterNodes` if any pod is re-scheduled on a different Node.
That was the point I raised in our slack discussion and my question is: is it a good idea to add a Pod watch in KCCM ? Is it the right place ?

Reply:
This has been adjusted at here: https://github.com/kubermatic/cloud-provider-kubevirt/pull/4/files#diff-846b22d59b07c412304bff178dae0b33a6c1a5fa4d5469a0bac8516b7205bb08R209